Sources/SpeziViews/Environment/Logger.swift

```diff
@@ -0,0 +1,31 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import OSLog
+import SwiftUI
+
```

Comment:
We generally add two spaces between an import statement and the first declaration.
```suggestion


```